app/model/Country.scala

```diff
@@ -0,0 +1,268 @@
+package model
+
+import scala.util.Try
+
+enum Country(
```

Comment:
I think we should stick to iso country codes only and let the presentation application translate into a more human readable name. 
For now we have to still support legacy things that might expect our particular wording for country names so it's just safer and easier to keep this hard coded list of countries. 
If we manage to completely drop the country names there's probably a standard library that we can use to get the ISO 3166-1 alpha 2 "short" codes